### ISSUE TITLE
implement mGBDT

### ISSUE THREAD

**Issue by Sandy4321** [catboost/catboost#755]
Problem: {
https://youtu.be/xgrUdncj1b4?t=2133
PyConBy: Anna Veronika Dorogush, CatBoost: gradient boosting for classification, regression, ranking
35:33
is it implemented how create embedding by catboost?
}
catboost version: {latest}
Operating System: {Windows}
CPU: {CPU}
# GPU: {GPU}

**Comment by Sandy4321** [comment]
what paper she meant about how to create embedding by using trees/GBM? There is no paper about thin on NeurIPS 2018https://nips.cc/Conferences/2018/Schedule?type=Poster  as mentioned in video?

**Comment by annaveronika** [comment]
https://arxiv.org/abs/1806.00007 - here is the paper. We plan to introduce implement this functionality in catboost.

**Comment by Sandy4321** [comment]
any progress to this?

**Comment by annaveronika** [comment]
We'll write here, when we will start working on that issue.

**Comment by kizill** [comment]
We've tested this approach and decided not to make it as an additional mode for training. But now CatBoost supports everything you need to build mGBDT just like it was done in original paper with XGBoost. So closing this issue.
